scilab_kernel/kernel.py

```diff
@@ -295,6 +296,13 @@ def _handle_svg(self, filename):
             im.data = self._fix_svg_size(im.data)
         except Exception:
             pass
+        try:
+            settings = self.plot_settings
+            print(settings)

```

Comment:
looks like a debug print, would you like to keep it ?

Reply:
You're right, fixed f0afba96b186d39400ca55f29a74341ab1b3b213